### PR TITLE
fix(nxext): fix app generator zone import

### PR DESCRIPTION
**Pull request by DominikPieper**

### CODE REVIEW

File: packages/angular/src/generators/application/application.ts [hunk 1]
```diff
@@ -69,7 +67,7 @@ export async function applicationGenerator(tree: Tree, options: Schema) {
   );
   const changes = applyChangesToString(source, [
     ...addImport(mainTsSourceFile, `import '@angular/compiler';`),
-    ...addImport(mainTsSourceFile, `zone.js';`),
+    ...addImport(mainTsSourceFile, `import 'zone.js';`),
```
Comment (Jordan-Hall) [inline packages/angular/src/generators/application/application.ts:70]: This isnt needed.  This actually needs removing now. Polyfill deals with it

**Comment by Jordan-Hall** [comment]
@DominikPieper is this ready to be merge in

**Comment by DominikPieper** [comment]
@Jordan-Hall yes. Just some simple things. I hate it to put too many different changes into one PR

**Comment by DominikPieper** [comment]
I try some more things out rn but I'd add another PR for that

**Pull request merged**